### PR TITLE
adjusted content to reflect csharp assignments

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Java Graded Assignment #4 Rubric (TechJobs, Persistent Edition)
```
Comment (codinglikeagirl42) [inline README.md]:
```suggestion
# C# Graded Assignment #4 Rubric (TechJobs, Persistent Edition)
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+This test project verifies the following:
+   * `AddJobViewModel` - Does it exist? How does it handle information?
+   * `Employer` and `Skill` - Do they contain the correct fields, correct accessors, correct validation annotations, constructors, and persistence.
+   * Controllers - Do controller methods route properly? Do they properly handle data inputs, different methods, and storing data?
+
+`TestTaskThree.Tests` contains the test class that verifies that students have set up the one-to-many relationship between jobs and skills properly.
+
+### Performance and Code Check:
+
+[Working Demo App](https://techjobs-persistent.launchcodetechnicaltraining.org/)
```
Comment (codinglikeagirl42) [inline README.md]: The working demo seems to be not be working -> slow loading and getting ->

Whitelabel Error Page
This application has no explicit mapping for /error, so you are seeing this as a fallback.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+1. Create a new Job from the form, selecting at least two skills. 
+
+   a. Once created, view the job table and verify that there is an ``employer_id`` column in it. 
+     
+   b. Confirm that a mapped table is present for ``job_skills`` and that at least two rows are now in it
+      for the job just created.
+
+## Submitting Your Work
+
+To submit your work please follow the [Submission Instructions](https://education.launchcode.org/csharp/assignments/hello-world.html#submitting-your-work)
```
Comment (codinglikeagirl42) [inline README.md]: Link does not work.

**Review by codinglikeagirl42 — CHANGES_REQUESTED** [review]
This was a brief review of the readme with anticipation of changes in Q1. At which time, the readme will need to be resubmitted for a formal QA review.

File: README.md (hunk 2 above)
Comment (codinglikeagirl42) [reply README.md]: Please add verbiage about why students may run into issues with the app.

Comment (gildedgardenia) [reply README.md]: Fixed!

File: README.md (hunk 3 above)
Comment (gildedgardenia) [reply README.md]: Fixed!

**Review by codinglikeagirl42 — APPROVED** [review]

**Pull request merged**